tests/scripts/test_pkgdev_commit.py

```diff
@@ -425,7 +425,7 @@ def commit():
 
         # initial package import
         repo.create_ebuild('cat/newpkg-0')
-        assert commit() == 'cat/newpkg: initial import'
+        assert commit() == 'cat/newpkg: new package, add 0'
```

Comment:
Adding a test for the overflowed case would be good too.

Reply:
Thanks for the reminder!